server/lemmy_db/src/site.rs

```diff
@@ -26,6 +28,8 @@ pub struct SiteForm {
   pub enable_downvotes: bool,
   pub open_registration: bool,
   pub enable_nsfw: bool,
+  pub icon: Option<Option<String>>,
+  pub banner: Option<Option<String>>,
```

Comment:
Diesel makes you do this for update forms... when you want to null out a column, you have to send `Some(None))`, since sending `None` means you just don't want to update that column.

Reply:
Best add this as a comment in the code.

Reply:
Done.